config/src/defaults.rs

```diff
@@ -60,6 +60,11 @@ pub trait Defaults {
         Duration::from_secs(30)
     }
 
+    /// Default period for discovering peers
```

Comment:
This could be a more accurate description:
```suggestion
    /// Default period for trying connecting to recently discovered peer addresses in search for quality peers
```
@mariocao and @girazoki please confirm I got it right.